test/specs/lib/AutoControlledComponent-test.js

```diff
@@ -95,6 +95,56 @@ describe('extending AutoControlledComponent', () => {
       wrapper
         .should.have.state(randomProp, props[randomProp])
     })
+
+    it('sets state for props passed as undefined by the parent', () => {
+      consoleUtil.disableOnce()
+
+      const props = makeProps()
+      const autoControlledProps = _.keys(props)
+
+      const randomProp = _.sample(autoControlledProps)
+      const randomValue = faker.hacker.phrase()
+
+      props[randomProp] = undefined
+
+      TestClass = createTestClass({ autoControlledProps, state: {} })
+      const wrapper = shallow(<TestClass {...props } />)
+
+      wrapper
+        .instance()
+        .trySetState({ [randomProp]: randomValue })
+
+      // not updated
+      wrapper
+        .should.have.state(randomProp, randomValue)
```

Comment:
Comment here says `// not updated`, but the assertion is that the `randomProp` was updated from `undefined` to the `randomValue`.  Is this correct?